UseCases/Sources/HTTPEndToEnd/InstrumentedHTTPClient.swift

```diff
@@ -41,7 +41,7 @@ struct InstrumentedHTTPClient {
     func execute(request: HTTPClient.Request, context: BaggageContext) -> EventLoopFuture<HTTPClient.Response> {
         var request = request
         self.instrument.inject(context, into: &request.headers, using: HTTPHeadersInjector())
-        context.logger.info("🌎 InstrumentedHTTPClient: Execute request")
+//        context.logger.info("🌎 InstrumentedHTTPClient: Execute request")
```

Comment:
should be possible to work now, uncomment?

Reply:
How? Wouldn't `context` have to be a `LoggingBaggageContextCarrier` in order for that to work?

Reply:
Ah, that's what I get for reviewing from a phone 😉 

Right that's good -- can you remove the comment then though?

Reply:
Replaced it with a `self.logger` so that the example at least continues to log.